config/state_machine/configs/00_moj/20_sar/20_sar_standard_workflow.yml

```diff
@@ -109,6 +109,10 @@
                   if: Workflows::Predicates#responder_is_member_of_assigned_team?
                   after_transition: Workflows::Hooks#reassign_user_email
 
+              rejected:
```

Comment:
This doesn't seem relevant for this ticket.
This also goes for the other changes to the state machine config.